elpy.el

```diff
@@ -315,6 +315,18 @@ edited instead. Setting this variable to nil disables this feature."
   :type '(repeat string)
   :group 'elpy)
 
+(defcustom elpy-test-django-runner-manage-command '("manage.py" "test"
+                                             "--noinput")
```

Comment:
Why is this on a new line? And if it is, why is it not indented as per normal Emacs Lisp rules (just use `TAB`)?